samples/msal-browser-samples/VanillaJSTestApp2.0/app/multi-tenant/authConfig.js

```diff
@@ -0,0 +1,68 @@
+const homeTenant = "ENTER_HOME_TENANT";
+const guestTenant = "ENTER_GUEST_TENANT"
+const baseAuthority = "https://login.microsoftonline.com"
+
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "ENTER_CLIENT_ID",
+        authority: `${baseAuthority}/${homeTenant}`,
+        multiTenantAccountsEnabled: true
```

Comment:
We should write some docs for this feature. Once those docs are written I would challenge whether we need a separate sample for this at all